src/directives/formio.js

```diff
@@ -48,7 +48,6 @@ module.exports = function() {
           'false': false
         };
 
-        var submission = $scope.submission || {data: {}};
```

Comment:
It wasn't necessary to have two separate variables track the submission object. The way we are doing it now is that the $watch looks at the $scope.submission.data and that is now used when determining the conditionals. I believe this fixes a lot of anomalies we were seeing with certain fields not working.